lib/websql/WebSQLDatabase.js

```diff
@@ -127,4 +185,45 @@ WebSQLDatabase.prototype.readTransaction = function (txnCallback, errorCallback,
   this._createTransaction(true, txnCallback, errorCallback, successCallback);
 };
 
+WebSQLDatabase.prototype.changeVersion = 
+  function (oldVersion, newVersion, txnCallback, errorCallback, successCallback) {
+  var self = this;
+  /* istanbul ignore next */
+  errorCallback = errorCallback || noop;
+  /* istanbul ignore next */
+  successCallback = successCallback || noop;
+  self.transaction(function (tx) {
+    tx.executeSql(getVersion.sql, getVersion.args, function (tx, res) {
```

Comment:
I would just have `getVersion` be a string and do `tx.executeSql(getVersion, [], ...` here.